pkg/cli/cmd/app/connections/display.go

```diff
@@ -64,71 +57,43 @@ func display(graph *applicationGraph) string {
 	}
 
 	for _, resource := range applicationResources {
-		if resource.Error != "" {
-			output.WriteString(fmt.Sprintf("Error: %s\n", resource.Error))
-			continue
-		}
-
-		output.WriteString(fmt.Sprintf("Name: %s (%s)\n", resource.Name, resource.Type))
+		output.WriteString(fmt.Sprintf("Name: %s (%s)\n", *resource.Name, *resource.Type))
 
 		if len(resource.Connections) == 0 {
 			output.WriteString("Connections: (none)\n")
 		} else {
 			output.WriteString("Connections:\n")
 			for _, connection := range resource.Connections {
-				if connection.To.Error != "" {
-					output.WriteString(fmt.Sprintf("  %s -> %s (%s)\n", connection.From.Name, "error", connection.To.Error))
+				connectionID, err := resources.Parse(*connection.ID)
+				if err != nil {
+					output.WriteString(fmt.Sprintf("Error: %s\n", err.Error()))
 					continue
 				}
 
-				// We format the connection differently depending on whether it's inbound or outbound.
-				if connection.From.ID == resource.ID {
+				connectionName := connectionID.Name()
+				connectionType := connectionID.Type()
+
+				if *connection.Direction == v20231001preview.DirectionOutbound {
 					// Outbound
-					output.WriteString(fmt.Sprintf("  %s -> %s (%s)\n", connection.From.Name, connection.To.Name, connection.To.Type))
+					output.WriteString(fmt.Sprintf("  %s -> %s (%s)\n", *resource.Name, connectionName, connectionType))
 				} else {
 					// Inbound
-					output.WriteString(fmt.Sprintf("  %s (%s) -> %s\n", connection.From.Name, connection.From.Type, connection.To.Name))
+					output.WriteString(fmt.Sprintf("  %s (%s) -> %s\n", connectionName, connectionType, *resource.Name))
 				}
 			}
 		}
 
-		if len(resource.Resources) == 0 {
+		if len(resource.OutputResources) == 0 {
 			output.WriteString("Resources: (none)\n")
 		} else {
 			output.WriteString("Resources:\n")
-			for _, resource := range resource.Resources {
-				if resource.Error != "" {
-					output.WriteString(fmt.Sprintf("Error: %s\n", resource.Error))
-					continue
-				}
-
-				link := makeHyperlink(resource)
-				if link == "" {
-					output.WriteString(fmt.Sprintf("  %s (%s: %s)\n", resource.Name, resource.Provider, resource.Type))
-				} else {
-					output.WriteString(fmt.Sprintf("  %s (%s: %s) %s\n", resource.Name, resource.Provider, resource.Type, link))
-				}
+			for _, resource := range resource.OutputResources {
+				output.WriteString(fmt.Sprintf("  %s (%s)\n", *resource.Name, *resource.Type))
 			}
 		}
 
 		output.WriteString("\n")
-	}
-
-	return output.String()
-}
 
-func makeHyperlink(resource outputResourceEntry) string {
-	// Just azure for now.
-	if resource.Provider != "azure" {
-		return ""
 	}
-
-	// format of an Azure portal URL:
-	//
```

Comment:
logged https://github.com/radius-project/radius/issues/6937 to bring in the URLs.

Reply:
Is there a reason why we can't keep this functionality for now?

Reply:
added it back..